sopht/simulator/immersed_body/cosserat_rod/cosserat_rod_forcing_grids.py

```diff
@@ -600,3 +601,547 @@ def _update_surface_grid_point_for_caps(self):
                 self.surface_point_rotation_angle_list[
                     rod_end_idx
                 ] = surface_point_angles
+
+
+class CosseratRodConstantTemperatureNodalForcingGrid(CosseratRodNodalForcingGrid):
+    def __init__(
+        self, grid_dim: int, cosserat_rod: ea.CosseratRod, rod_temperature: float
+    ) -> None:
+        super().__init__(grid_dim, cosserat_rod)
+
+        self.rod_temperature = rod_temperature
+
+        self.velocity_field = np.ones((self.num_lag_nodes)) * self.rod_temperature
+
+        self.compute_lag_grid_position_field()
+        self.compute_lag_grid_velocity_field()
```

Comment:
These should probably be abstracted somehow in a later fix considering we are doing this in every initialization even when not necessary. But okay for now.